services/ingestion/etl/types/Project.js

```diff
@@ -56,6 +56,11 @@ type Budget = {
   total_cost: BudgetItem,
 };
 
+type Coordinates = {
```

Comment:
Too pity we can't easily re-use the types provided by `flow-geojson`. There are [some exports](https://github.com/bigslycat/flow-geojson/blob/a8b38476a481136590f3310cf8a9d6188625b182/src/GeoJSON.js#L56) but it's quite hard to see whether they would be more useful in comparison to specifying the properties like this.

Reply:
I could have used `Coord2DStrict` but you have to know the order `[Longitude, Latitude]`.  Using an object with `lat` and `lon` is far more explicit and less error-prone